.github/workflows/main.yml

```diff
@@ -1,6 +1,12 @@
 name: Node CI
 
-on: [push]
+on:
+  push:
+    branches:
+      - master
```

Comment:
limiting to only run on pushes to master to avoid duplicate runs: 
![image](https://user-images.githubusercontent.com/1404810/133441267-6651be72-d174-4e27-9066-ed0a232e1e9e.png)